src/App.css

```diff
@@ -1,7 +1,235 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@600&display=swap');
+
+#root {
+  margin: 0;
+  padding: 0 0 8%;
+  border-radius: 4px;
+  box-shadow: 0 0 60px 0 rgb(0 0 0 / 5%);
+  background-color: #fafafa;
```

Comment:
- [ ] Dry principles of CSS code are encouraged here, you have this background color being applied in another class in line 31 causing repetition. Kindly follow DRY principles by creating a class with the background color that can be reused in the components, [refer to this tutorial for more help](https://speckyboy.com/good-bad-css-practices/)